src/magic_uv/op/align_uv_cursor.py

```diff
@@ -33,20 +33,15 @@
 from ..utils.property_class_registry import PropertyClassRegistry
 from ..utils import compatibility as compat
 
-
 def _is_valid_context(context):
     # 'IMAGE_EDITOR' and 'VIEW_3D' space is allowed to execute.
     # If 'View_3D' space is not allowed, you can't find option in Tool-Shelf
-    # after the execution
```

Comment:
I think this change makes the CI fail.
Could you delete the spaces following this comment?